integration-tests/src/test/java/org/apache/druid/tests/indexer/ITKafkaIndexingServiceTest.java

```diff
@@ -150,6 +150,8 @@ public void testKafka()
       LOG.info("supervisorSpec name: [%s]", INDEXER_FILE);
       Properties consumerProperties = new Properties();
       consumerProperties.put("bootstrap.servers", config.getKafkaInternalHost());
+      consumerProperties.put("auto.offset.reset", "none");
```

Comment:
Can we use `KafkaConsumerConfigs` here, too?

Reply:
yeah, will change